.github/workflows/checkers.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: JustinBeckwith/linkinator-action@v1.10.3
+    - uses: JustinBeckwith/linkinator-action@v1.10.4
```

Comment:
```suggestion
    - uses: JustinBeckwith/linkinator-action@v1
```

Reply:
The GitHub app isn't letting me do that. I'll do something like what you suggest in a future change.